workers/pyproject.toml

```diff
@@ -14,24 +14,26 @@ keywords = [
   "quality",
   ]
 
+[[tool.poetry.source]]
+name = "PyPI"
+priority = "primary"
+
 [[tool.poetry.source]]
 name = "gistools-ohsome-machine-learning-models"
 url = "https://gitlab.gistools.geog.uni-heidelberg.de/api/v4/groups/277/-/packages/pypi/simple"
-secondary = true
+priority = "supplemental"
 
 [tool.poetry.dependencies]
 python = "^3.10"
 geojson = "^2.5.0"
 fastapi = "^0.92.0"
-requests = "^2.25.0"
-dacite = "^1.6.0"
+requests = "^2.31.0"
 PyYAML = "^6.0"
 toml = "^0.10.2"
 httpx = "^0.23.0"
 asyncpg = "^0.27.0"
 vcrpy = "^4.1.1"
 python-dateutil = "^2.8.2"
-Jinja2 = "^3.1.2"
```

Comment:
Why don't we need Jinja2 anymore? It is still in use in the code.

Reply:
Ups. Good that you spotted this. I dropped the commit.